FunctionHomework/test/FahrenheitToCelsiusTest.java

```diff
@@ -0,0 +1,16 @@
+
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FahrenheitToCelsiusTest {
+    FahrenheitToCelsius v = new FahrenheitToCelsius();
```

Comment:
change is a static method, the recommended way to call it is FahrenheitToCelsius.change(....

no need to do new Fah... for static methods